snapwell/snapwell_main.py

```diff
@@ -134,81 +136,45 @@ def load_config(self, args):
         snap_conf = args.config
 
         if args.owc_offset:
-            snap_conf.setOwcOffset(args.owc_offset)
+            snap_conf.owc_offset = args.owc_offset
         if args.output:
             outpath = path.abspath(args.output)
             if path.isfile(outpath):
                 self.exit_with_usage(
                     "Output path is an existing file. Delete it or choose a different output path."
                 )
-            if not path.exists(outpath):
-                makedirs(outpath)
-            snap_conf.setOutput(outpath)
+            snap_conf.output_dir = outpath
         if args.overwrite:
-            snap_conf.setOverwrite(args.overwrite)
+            snap_conf.overwrite = args.overwrite
         if args.owc_definition:
-            snap_conf.setOwcDefinition(args.owc_definition)
+            snap_conf.owc_definition = args.owc_definition
         if args.owc_offset:
-            snap_conf.setOwcOffset(args.owc_offset)
+            snap_conf.owc_offset = args.owc_offset
         if args.delta:
-            snap_conf.setDeltaZ(args.delta)
+            snap_conf.delta_z = args.delta
         return snap_conf
 
-    def load_wellpaths(self, config):
-        logging.info("Loading %d wells", len(config))
-
-        wellpaths = []
-        for i in range(len(config)):
-            fname = config.filename(i)
-            logging.info(fname, end=" ... ")
-            wp = config.getWellpath(i)
-            logging.info("(%d points, %d logs)", len(wp), len(wp.headers()))
-            wellpaths.append(wp)
-            logging.info("done")
-        return wellpaths
-
     def load_restart_file(self, config):
-        logging.info("Loading restart %s", config.restartFile())
-        restart = None
-        restartFile = config.restartFile()
+        logging.info("Loading restart %s", config.restart_file)
         try:
-            restart = config.getRestart()
+            return config.restart
         except Exception as err:
-            logging.warning("supplied RESTART file not loaded: %s", err)
-
-        if not restart:
-            if not path.isfile(restartFile):
-                self.exit_with_usage("Missing restart: No such file %s." % restartFile)
-            else:
-                self.exit_with_usage(
-                    "Missing restart: Failed to read restart file %s." % restartFile
-                )
-        return restart
+            logging.error("supplied RESTART file not loaded: %s", err)
 
     def load_grid_file(self, config):
-        logging.info("Loading grid %s", config.gridFile())
-        grid = None
-        gridFile = config.gridFile()
+        logging.info("Loading grid %s", config.grid_file)
         try:
-            grid = config.getGrid()
+            return config.grid
         except Exception as err:
```

Comment:
Can you make this exception explicit instead of blanket?

Reply:
Maybe, I will look at the Grid constructor from ecl.

Reply:
Do you know if any other `Exception` than `IOException` that would need to be handled?

Reply:
Not as far as I know, if this fails for any other reason I think it will `util_abort`, which is non-catchable.

Reply:
Only catch `IOError` now.